src/plugins/intel_cpu/src/nodes/stft.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "stft.h"
+
+#include "nodes/common/cpu_memcpy.h"
+#include "openvino/core/parallel.hpp"
+#include "openvino/core/type.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/stft.hpp"
+#include "openvino/reference/stft.hpp"
+
+namespace ov {
+namespace intel_cpu {
+namespace node {
+
+bool STFT::isSupportedOperation(const std::shared_ptr<const ov::Node>& op, std::string& errorMessage) noexcept {
+    try {
+        if (op->get_type_info() != op::v15::STFT::get_type_info_static()) {
+            errorMessage = "Only STFT operation from the opset15 is supported by the CPU plugin.";
+            return false;
+        }
+    } catch (...) {
+        return false;
+    }
+    return true;
+}
+
+STFT::STFT(const std::shared_ptr<ov::Node>& op, const GraphContext::CPtr& context)
+    : Node(op, context, NgraphShapeInferFactory(op, PortMask(2, 3))) {
+    std::string errorMessage;
+    if (!isSupportedOperation(op, errorMessage)) {
+        THROW_CPU_NODE_ERR(errorMessage);
+    }
+
+    const auto stft_op = as_type_ptr<op::v15::STFT>(op);
+    m_transpose_frames = stft_op->get_transpose_frames();
+
+    m_is_frame_size_const = is_type<op::v0::Constant>(stft_op->get_input_node_ptr(FRAME_SIZE_IDX));
+    m_is_frame_step_const = is_type<op::v0::Constant>(stft_op->get_input_node_ptr(FRAME_STEP_IDX));
+}
+
+void STFT::getSupportedDescriptors() {
+    if (getParentEdges().size() != 4) {
+        THROW_CPU_NODE_ERR("STFT has incorrect number of input edges.");
+    }
+    if (getChildEdges().empty()) {
+        THROW_CPU_NODE_ERR("STFT has incorrect number of output edges.");
+    }
+}
+
+void STFT::initSupportedPrimitiveDescriptors() {
+    if (!supportedPrimitiveDescriptors.empty())
+        return;
+
+    const auto& dataPrecision = getOriginalInputPrecisionAtPort(DATA_IDX);
+    if (!dataPrecision.is_real()) {
```

Comment:
```
if (!one_of(dataPrecision , f32, fp16, bf16)) { // leave f32 if reference could not be extended for other now
    dataPrecision = f32;
}
```

Reply:
Currently only f32 is supported by ref.
Thanks, applied.